Reverse_array_in_groups.js

```diff
@@ -0,0 +1,27 @@
+const SIZE_OF_ARRAY = +process.argv[2];
+const SUB_ARRAY = +process.argv[3];
+const STRING_ARRAY = process.argv[4].trim().split(',')
+const NUMBER_ARRAY = STRING_ARRAY.map((ele) => { return +ele })
```

Comment:
no need to write return use (ele) => (+ele);

Reply:
Noted.